content/10.blackbox.md

```diff
@@ -1,25 +1,26 @@
 ## Tip 8: Deep learning models can be made more transparent {#blackbox}
 
 While model interpretability is a broad concept, in much of the machine learning literature (including in our guidelines), it refers to the ability to identify the discriminative features that influence or sway the predictions.
-In certain cases, the goal behind interpretation is to understand the underlying data generating processes while in other cases the goal is to understand why a model made the prediction that it did for a specific example or set of examples.
-Machine learning models vary widely in terms of interpretability: some are fully transparent while others are considered to be "black-boxes" that make predictions with little ability to examine why.
-Logistic regression and decision tree models are generally considered interpretable, while deep neural networks are often considered among the most difficult to interpret because they can have many parameters and non-linear relationships. 
+In certain cases, the goal behind interpretation is to understand the underlying data generating processes and biological mechanisms [@doi.org/10.3390/biom10030454]. In other cases, the goal is to understand why a model made the prediction that it did for a specific example or set of examples.
+Machine learning models vary widely in terms of interpretability: some are fully transparent while others are considered "black-boxes" that make predictions with little ability to examine why.
+Logistic regression and decision tree models are generally considered as very interpretable [@doi:10.1007/978-1-4939-7756-7_16]. In contrast, deep neural networks are often considered among the most difficult to interpret because they can have many parameters and non-linear relationships. 
```

Comment:
```suggestion
Logistic regression and decision tree models are generally considered interpretable [@doi:10.1007/978-1-4939-7756-7_16].
In contrast, deep neural networks are often considered among the most difficult to interpret because they can have many parameters and non-linear relationships. 
```